Tests/ApolloCodegenTests/CodeGeneration/Templates/SelectionSet/SelectionSetTemplateTests.swift

```diff
@@ -6037,6 +6037,153 @@ class SelectionSetTemplateTests: XCTestCase {
     expect(actual).to(equalLineByLine(expected, atLine: 1, ignoringExtraLines: true))
   }
 
+  func test__render_conditionalFragmentOnQueryRoot__rendersRootEntityType() throws {
```

Comment:
These are the tests for the actual bug fix. The rest of the test changes are just refactoring for other minor API changes.